lib/pipa-gateway.js

```diff
@@ -146,41 +154,52 @@ PipaGateway.prototype.__route__handler = {
   *   Call multiple services in one time.
   */
 
-  parallel: function (domain, httpMethod, config) {
-    return function (req, res, next) {
+  parallel: function(domain, httpMethod, config) {
+    return function(req, res, next) {
       var worker = {};
       var method = httpMethod.toUpperCase();
 
-      config.services.forEach(function (item) {
-        worker[item.name] = function (cb) {
+      config.services.forEach(function(item) {
+        worker[item.name] = function(cb) {
           var opts = {
             method: method,
             uri: helper.generateUrl(domain, item.url, req.params),
             headers: helper.removeDefaultHeaders(req.headers),
             json: true
           };
 
-          if (method === 'GET')
-            opts.qs = req.query;
-          else
-            opts.body = req.body;
+          if (method === "GET") opts.qs = req.query;
+          else opts.body = req.body;
 
-          request(opts, function (err, response, body) {
+          request(opts, function(err, response, body) {
             if (err)
-              return cb(null, { statusCode: err.statusCode || 500, err: err.response.body });
+              return cb(null, {
+                statusCode: err.statusCode || 500,
+                err: err.response.body
+              });
 
             cb(null, { statusCode: response.statusCode || 200, body: body });
           });
         };
       });
 
-      async.parallel(worker, function (err, result) {
+      async.parallel(worker, function(err, result) {
+        var statusCode = 200;
```

Comment:
Since the value of `statusCode` variable is not being used, you can change it to `var statusCode;` instead.